packages/fxa-settings/src/components/Security/index.test.tsx

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import Security from '.';
+
+const { findByText, findAllByText } = screen;
```

Comment:
I'm not sure these should be global for the entire file, or at least we haven't destructured these out of `screen` before - we've typically just used `screen.whatevermethod` because that allows you to see the available methods in VSCode (after typing `screen.`), or we use `const { findByText } = render(<SomeComponent`. We discussed this a bit [here](https://github.com/mozilla/fxa/pull/5690#discussion_r443627367) if you want to read more.